lib/src/injector_base.dart

```diff
@@ -78,13 +83,18 @@ class Injector {
    *
    * You get the same instance of your dependency every time!
    *
+   * [force] => Override the existing dependency
```

Comment:
Can we quickly streamline the documentation style? using `///` or `/**/`, not mixed?